pkg/cmd/connect/connect.go

```diff
@@ -25,10 +25,10 @@ var ConnectCmd = &cobra.Command{
 	Short: `Connect resources for LitmusChaos Execution plane.
 		Examples:
 		#connect a Chaos Infrastructure
-		litmusctl connect chaos-infra --name="new-chaos-infra" --non-interactive
+		litmusctl connect chaos-infra --name="new-chaos-infra" --non-interactive --environment-id="my-environment-id"
```

Comment:
No need to add incase of interactive mode, user will see a prompt to enter the environment-id incase of interactive mode